### ISSUE TITLE
multiple choice bug tasks , when adding a watcher to the tasks all the tasks are deleted from the list 

### ISSUE THREAD

**Issue by yaronmir** [linnovate/root#1288]
multiple choice bug tasks , when adding a watcher to the tasks all the tasks are deleted from the list , and only after creating a new task, that all the other tasks re-appear

**Comment by abrahamos** [comment]
Failed to reproduce

**Comment by manmorc** [comment]
none-reproducible
